modules/dnn/src/dnn_utils.cpp

```diff
@@ -196,7 +196,7 @@ void blobFromImagesWithParamsImpl(InputArrayOfArrays images_, Tmat& blob_, const
                 int bottom = size.height - top - rh;
                 int left = (size.width - rw)/2;
                 int right = size.width - left - rw;
-                copyMakeBorder(images[i], images[i], top, bottom, left, right, BORDER_CONSTANT);
+                copyMakeBorder(images[i], images[i], top, bottom, left, right, BORDER_CONSTANT, param.borderValue);
```

Comment:
> `images[i], images[i]`

Operation is not in-place.
We should not use the same argument for `src` and `dst` - this is anti-pattern and bad practice.